lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -91,8 +95,8 @@
       <li class="action action-share">
         <a class="action-linkedin-profile" target="_blank" href="${cert_status['linked_in_url']}"
          title="${_('Add Certificate to LinkedIn Profile')}"
-         data-course-id="${unicode(course_overview.id)}"
-         data-certificate-mode="${cert_status['mode']}"
+         data-course-id="${course_overview.id | n, unicode}"
+         data-certificate-mode="${cert_status['mode'] | n, js_escaped_string}"
```

Comment:
@robrap Can you clarify here? I do feel compelled to agree with @rlucioni, but neither of us are sure.